maven-wrapper-distribution/src/resources/mvnw

```diff
@@ -192,7 +192,8 @@ else
       wrapperUrl="https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/@@project.version@@/maven-wrapper-@@project.version@@.jar"
     fi
     while IFS="=" read key value; do
-      case "$key" in (wrapperUrl) wrapperUrl="$value"; break ;;
+      # Remove invalid line endings from value
+      case "$key" in (wrapperUrl) wrapperUrl="${value%$'\r'}"; break ;;
```

Comment:
...and you are sure this runs on a POSIX only shell?

Reply:
My testing was on Windows 10, Windows 2016 server, and RHEL 8.  So I don't know for sure but the logic there is posix compliant per multiple sources such as [here](https://www.ibm.com/docs/en/aix/7.1?topic=shell-parameter-substitution-in-korn-posix) and [here](https://unix.stackexchange.com/questions/389209/what-kinds-of-string-interpolation-does-posix-sh-support).

Reply:
@michael-o When you get a chance, can you check in on this again?  Thanks.

Reply:
Not this month. I have basically no interest in the wrapper since I consider it more or less pointless.